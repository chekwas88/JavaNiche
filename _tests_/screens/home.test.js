import React from 'react';
import { shallow } from 'enzyme';
import HomeScreen from '../../src/screens/Home';

describe('Test for HomeScreen component', () => {
  const wrapper = shallow(<HomeScreen />);
  test('should test for login ', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
describe('Test for proper rendering of home screen', () => {
  it('should call onPress', () => {
    const wrapper = shallow(<HomeScreen />);
    expect(
      wrapper
        .find('ScrollViewMock')
        .find('DetailsList')
        .exists()
    ).toBe(true);
  });
});
