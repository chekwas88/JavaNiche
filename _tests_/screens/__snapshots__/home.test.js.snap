// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for HomeScreen component should test for login  1`] = `ShallowWrapper {}`;
